test/test_arcsine.cpp

```diff
@@ -292,6 +301,16 @@ void test_spots(RealType)
     BOOST_CHECK_CLOSE_FRACTION(pdf(arcsine_01, static_cast<RealType>(1) - tolerance),
       1 /(sqrt(tolerance) * boost::math::constants::pi<RealType>()), 2 * tolerance); //
 
+    // Log PDF
+    BOOST_CHECK_CLOSE_FRACTION(logpdf(arcsine_01, 0.000001), static_cast<RealType>(5.7630258931329868780772138043668005779060097243996L), tolerance);
```

Comment:
One thing I worry about these tests is that if they fail at some later date, it's not clear why (say) `5.7630258931329868780772138043668005779060097243996L` is the correct value. Is there a property that can be tested instead?